### PR TITLE
Iframe

### PR DESCRIPTION
**Pull request by Developerayo**
created an iframe for the sidebar registration

### CODE REVIEW

**Comment by ooade** [comment]
Where does register on the "about" page lead to?

**Comment by Developerayo** [comment]
@ooade Right now where, but it would be pointing out to the registration form for our main event on 29th next month.

**Comment by kingisaac95** [comment]
I don't think we need `mission.html` and `sponsors.html`

**Comment by kingisaac95** [comment]
Also, I've been trying to get the iframe to display, but I can't seem to get to that state. How does that work @Developerayo?

**Comment by ooade** [comment]
Asides @kingisaac95's last question, I was wondering why a registration link isn't on the landing page?

**Comment by kingisaac95** [comment]
Pushed a commit to fix copyright and remove unnecessary nav items.

**Comment by Developerayo** [comment]
> I don't think we need `mission.html` and `sponsors.html`

Hmm, @ooade_ what do you think?

**Comment by Developerayo** [comment]
> Also, I've been trying to get the iframe to display, but I can't seem to get to that state. How does that work @Developerayo?

Which iframe? the sidebar?

**Comment by kingisaac95** [comment]
> Which iframe? the sidebar?

Yes

**Comment by Developerayo** [comment]
> Asides @kingisaac95's last question, I was wondering why a registration link isn't on the landing page?

We do not have a registration page yet with complete details but we can put in the link now though 

@ooade could you help insert the link into all the registration buttons [https://eventprime.co/e/react-present-and-future-a-hands-on-approach](https://eventprime.co/e/react-present-and-future-a-hands-on-approach)

**Comment by Developerayo** [comment]
> > Which iframe? the sidebar?
> 
> Yes

I remove that feature in all the pages because we will be using a software for registration that we are building and working with at Fremer named eventprime, but event prime does not have a scroll feature eg. `scrolling="yes"` so the form is not scrollable in that iframe and it cannot be edited that's the issue with the iframe

File: index.html [hunk 1]
```diff
@@ -173,9 +135,9 @@ <h2>The unStack approach</h2>
             </div>
             <div class="content-column">
                 <div class="text">
-                    <strong>unStack</strong> is a community of passionate engineers helpers other engineers grow their craft.
-                    <strong>unStack</strong> is mostly focused on <strong>hands-on technical workshops and conferences</strong> around web and mobile development.
-                    We'll be focusing on helping the audience gain in-dept understanding of any chosen topic area,
+                    <strong>unStack</strong> is a community of passionate developers, designers, and other technical talents.
+                    <strong>unStack</strong> is mostly focused on <strong>hands-on technical workshops and conferences</strong> around the 21st-century technology.
```
Comment (kingisaac95) [inline index.html]:
```suggestion
                    <strong>unStack</strong> is mostly focused on <strong>hands-on technical workshops and conferences</strong> around web, mobile and other emerging technologies.
```

**Comment by Developerayo** [comment]
Oh wow, I just fixed it and it works "the iframe".

**Comment by kingisaac95** [comment]
> Oh wow, I just fixed it and it works "the iframe".
> 
> Should I bring back the sidebar feature?

What's the sidebar for again? If we're using event prime for now, then the sidebar can wait.

**Comment by Developerayo** [comment]
> > Oh wow, I just fixed it and it works "the iframe".
> > Should I bring back the sidebar feature?
> 
> What's the sidebar for again? If we're using event prime for now, then the sidebar can wait.

The white down arrow at the top right hand side

**Pull request merged**